### ISSUE TITLE
GROW 🌱

### ISSUE THREAD

**Issue by lauriehimmel** [LouisPino/Kindr#58]
**What is one area in your code that can be improved or what is already great, but could be enhanced? Why?**

There is a lot of room for cleaning up our code, specifically for using components and props more/more intentionally. Much of our code is found in pages, and there are areas where code is duplicated or accessed in ways it wouldn't need to be if we had used more components.

**Comment by lauriehimmel** [comment]
Some additional grows post-code review:
- add an info page with how to use the app
- update text that says "challenge"
- reinstate a default profile name
- connect google back to auth0
- deed page updates on image upload
- update "Created by" name if username updates

ideas from peers:
- add what time deed was added
- undo completing a deed in case it was accidental
- clearer page delineation so users are clear what page they're on

**Comment by krabecb** [comment]
I agree, definitely clean up where you can, especially if there are duplicates! Make sure to keep spacing consistent and remove any console logs that are not being used! 

Love the list of additional grows, I think these are excellent icebox features! Keep us posted on your progress!
